src/components/header/index.js

```diff
@@ -28,7 +28,7 @@ class Header extends React.Component {
 
     onSelectChange = e => {
         const { teams, getSelectedTeam } = this.props;
-        const selectedTeam =  teams.filter(team => team.teamId === e.target.value).pop();
+        const selectedTeam = teams.filter(team => team.teamId === e.target.value).pop();
```

Comment:
Try to avoid using `pop()` as it changes the array itself and can create undesirable results. Use `find` instead of `filter`:

```
const selectedTeam = teams.find(team => team.teamId === e.target.value);
```